xarray/tests/test_dataarray.py

```diff
@@ -2518,6 +2518,40 @@ def test_combine_first(self):
                              [('x', ['a', 'b', 'd']), ('y', [-1, 0])])
         self.assertDataArrayEqual(actual, expected)
 
+    def test_sortby(self):
+        da = DataArray([[1, 2], [3, 4]],
+                       [('x', ['b', 'a']), ('y', [1, 0])])
+
+        expected = DataArray([[4, 3], [2, 1]],
+                             [('x', ['a', 'b']), ('y', [0, 1])])
+
+        actual = da.sortby(['x', 'y'])
```

Comment:
It would be good to include a test for a single sorted, too, just `da.sortby('x')`. Likewise `da.sortby(dax)`.